.travis.yml

```diff
@@ -22,6 +24,7 @@ services:
 
 before_install:
   - ./.travis/preinstall
+  - sudo apt get -y install python3-gi python-gi gir1.2-gtk-3.0 python-gi-cairo python3-gi-cairo
```

Comment:
This seems to be a duplicate of the packages from the preinstall script.

Reply:
gone now